core/block_svg.js

```diff
@@ -390,6 +390,28 @@ Blockly.BlockSvg.prototype.moveToDragSurface_ = function() {
   this.workspace.blockDragSurface_.setBlocksAndShow(this.getSvgRoot());
 };
 
+/**
+ * Move a block to a position.
+ * @param {goog.math.Coordinate} xy The position to move to in workspace units.
+ */
+Blockly.BlockSvg.prototype.moveTo = function(xy) {
```

Comment:
This is duplicating most of the code in moveBy. Unless there's a reason not to can we make one of them just convert the units and then call the other?

Reply:
I added a todo to the cleanup for keyboard nav list but would prefer to keep that separate from the keyboard nav to develop move.